src/Text/Pandoc/Readers/JATS.hs

```diff
@@ -273,24 +274,32 @@ parseBlock (Elem e) =
                       let headRowElements = case filterChild (named "thead") e' of
                                       Just h -> maybe [] parseElement (filterChild isRow h)
                                       Nothing -> []
+                      let bodyRowElements bodyElement = 
+                            map parseElement $ filterChildren isRow bodyElement
+
                       -- list of list of body cell elements 
-                      let bodyRowElements = case filterChild (named "tbody") e' of
-                                      Just b -> map parseElement $ filterChildren isRow b
-                                      Nothing -> map parseElement $ filterChildren isRow e'
+                      let multipleBodyRowElements = 
+                            map bodyRowElements $ filterChildren (named "tbody") e'
```

Comment:
Here we switch `filterChild` (parsing zero or one `tbody`s) to `filterChildren` (parsing zero or many `tbody`s)